cmd/boulder-va/gsb.go

```diff
@@ -78,39 +89,62 @@ func (sb gsbAdapter) IsListed(url string) (string, error) {
 	return "", nil
 }
 
+// gsbLogAdapter adapts a blog.Logger to the io.Writer interface used by the
+// Google safebrowsing client for a logger. All messages written to the Writer
+// by the library will be adapter to the logger's Info method.
+type gsbLogAdapter struct {
+	log blog.Logger
+}
+
+func (a gsbLogAdapter) Write(b []byte) (int, error) {
+	a.log.Info(string(b))
+	return len(b), nil
+}
+
 // newGoogleSafeBrowsingV4 constructs a va.SafeBrowsing instance using the new
 // Google upstream Safe Browsing version 4 client.
-func newGoogleSafeBrowsingV4(gsb *cmd.GoogleSafeBrowsingConfig) va.SafeBrowsing {
+func newGoogleSafeBrowsingV4(gsb *cmd.GoogleSafeBrowsingConfig, logger blog.Logger) (va.SafeBrowsing, error) {
 	// If there is no GSB configuration, don't create a client
 	if gsb == nil {
-		return nil
+		return nil, nil
 	}
 	if err := configCheck(gsb); err != nil {
-		cmd.FailOnError(err, "unable to create new safe browsing v4 client")
+		return nil, err
 	}
+
+	// Create the DB file if it doesn't exist
```

Comment:
I'm not sure where I got the idea it balked if the DB File wasn't created - probably confused myself when I discovered that (unsuprisingly) it balks when given a directory instead of a file.

I removed the "create if doesn't exist" code and confirmed everything worked as expected.